mods/mc_worldmanager/commands.lua

```diff
@@ -245,6 +250,72 @@ commands["define"] = {
         Realm:Restore(newRealm)
     end }
 
+commands["privs"] = {
+    func = function(name, params)
+        local operation = tostring(params[1])
+        local realmID = tonumber(params[2])
+        local privilege = tostring(params[3])
+
+        if (operation == nil or operation == "") then
+            return false, "Incorrect parameter... Missing realm privilege operation. Usage: realm privs [<grant> | <revoke>] <realmID> <privilege>"
+        end
+
+        if (operation == "help") then
+            return true, "Usage: 'realm privs [grant | list | revoke] <realmID> <privilege>'; 'realm privs help' for help."
+        end
+
+        if (realmID == nil or realmID == "") then
```

Comment:
`tonumber(nil)` on line [256](https://github.com/ubc-minetest-classroom/Minetest_Classroom/pull/100/files/1fb230731a440999d398ef2da907cc6f43b83610#diff-438bd03cc76fd81ad9448a1f1d70fcb5c719286e2841b32e95fb546e6d172d73R256) produces `nil`, so no patch is needed here